Source/EVEClassLibrary/ConsoleMain.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Runtime;
+using System.Reflection;
+using System.IO;
+using Newtonsoft;
+
+namespace EVEClassLibrary
+{
+    /// <summary>
+    /// Main Console Class, Handles the Basic User Input
+    /// </summary>
+    public class ConsoleMain
+    {
+        /// <summary>
+        /// Initializing Console
+        /// </summary>
+        public static void init()
+        {
+            Creds();
+            Spacer(2);
+            StartInfo();
+            ParseCommand();
+        }
+
+        /// <summary>
+        /// Credits of the Library
+        /// </summary>
+        public static void Creds()
+        {
+            System.Type type = typeof(ConsoleMain);
+
+            string Author =  type.Assembly.GetCustomAttribute<AssemblyCopyrightAttribute>().Copyright;
+            Console.Title =  type.Assembly.GetCustomAttribute<AssemblyProductAttribute>().Product;
+            string version = type.Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>().Version.ToString();
+
+            
+            Console.ForegroundColor = ConsoleColor.DarkGray;
+            Console.WriteLine($"written by {Author}");
+            Console.WriteLine($"Beta {version}");
+            Console.WriteLine("DO NOT REDISTRIBUTE");
+            Console.WriteLine("Work in progress");
+            Console.ForegroundColor = ConsoleColor.White;
+        }
+
+        /// <summary>
+        /// Pseudo Menu
+        /// </summary>
+        public static void StartInfo()
+        {
+            Console.WriteLine("Available Commands:");
+            Console.WriteLine("   [M]arketinformations");
+            Console.WriteLine("   [T]ypeinformation");
+            Console.WriteLine("   [C]lear Screen");
+            Spacer(2);
+        }
+
+
+        /// <summary>
+        /// Spacing Method, created empty lines with a simple command
+        /// </summary>
+        /// <param name="space"> the int representation of the lines to be kept empty</param>
+        public static void Spacer(int space)
+        {
+            for (int i = 0; i < space; i++)
+            {
+                Console.WriteLine();
+            }
+        }
+
+        /// <summary>
+        /// Clears the Screen
+        /// </summary>
+        public static void ClearScreen()
+        {
+            Console.Clear();
+        }
+
+        /// <summary>
+        /// Parses the Key presses for the Main Menu
+        /// </summary>
+        public static void ParseCommand()
+        {
+            ConsoleKeyInfo pressed = Console.ReadKey(true);
+            switch (pressed.Key)
+            {
+
+                case ConsoleKey.C:
+                    ClearScreen();
+                    StartInfo();
+                    break;
+
+                case ConsoleKey.M:
+                    ESIRunner runner = new ESIRunner();
+                    Console.WriteLine(runner.MarketUrl());
+                    break;
+
+                case ConsoleKey.T:
+                    //implement Type Checker
```

Comment:
Make this a //TODO-Comment so that VS can pick this up as TODO for the warningsection as well as the TODO-items section

Reply:
gotcha, will add TODO for the comments